### PR TITLE
Fix udelay issues that can make duration slightly too short

### PR DESCRIPTION
**Pull request by jwerner-chromium**
This fixes a minor problem we noticed that can result in udelay() calls delaying for slightly less time than the programmer intended.

### CODE REVIEW

**Comment by danh-arm** [comment]
jenkins: test this please

**Comment by davidcunado-arm** [comment]
jenkins: test this please

File: drivers/delay_timer/delay_timer.c [hunk 1]
```diff
@@ -30,7 +30,8 @@ void udelay(uint32_t usec)
 
 	start = ops->get_timer_value();
 
-	total_delta = (usec * ops->clk_div) / ops->clk_mult;
+	/* Add an extra tick to avoid delaying less than requested. */
+	total_delta = div_round_up(usec * ops->clk_div, ops->clk_mult) + 1;
```
Comment (davidcunado-arm) [inline drivers/delay_timer/delay_timer.c]: Need to include utils_def.h.

Comment (jwerner-chromium) [reply drivers/delay_timer/delay_timer.c]: Thanks, updated.

**Comment by davidcunado-arm** [comment]
jenkins: test this please

**Comment by davidcunado-arm** [comment]
jenkins: test this please

**Comment by davidcunado-arm** [comment]
check fail - looks like a false positive; static analyser getting confused with the __extension__ I guess.

**Pull request merged**